Sources/BreezeDynamoDBService/BreezeDynamoDBService.swift

```diff
@@ -76,10 +77,9 @@ public actor BreezeDynamoDBService: BreezeDynamoDBServing {
         logger.info("DBManager is ready.")
     }
     
-    /// Returns the BreezeDynamoDBManaging instance.
-    public func dbManager() async -> BreezeDynamoDBManaging {
-        logger.info("Starting DynamoDBService...")
-        return self.dbManager
+    public func run() async throws {
+        try await gracefulShutdown()
+        try await onGracefulShutdown()
```

Comment:
Are you shutting down immediately at run() ?  Does it work ? Why ?

Reply:
The function `try await gracefulShutdown()` waits until graceful shutdown is triggered as documented in ServiceLifecycle

https://swiftpackageindex.com/swift-server/swift-service-lifecycle/main/documentation/servicelifecycle/gracefulshutdown()